packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -273,6 +284,93 @@ const SelectedEntriesTableContent = ({
 
 const BoldChunk = (chunks: React.ReactNode) => <Typography fontWeight="bold">{chunks}</Typography>;
 
+/* -------------------------------------------------------------------------------------------------
+ * PublicationStatusSummary
+ * -----------------------------------------------------------------------------------------------*/
+
+interface PublicationStatusSummaryProps {
+  count: number;
+  icon: React.ReactNode;
+  message: string;
+}
+
+const PublicationStatusSummary = ({ count, icon, message }: PublicationStatusSummaryProps) => {
+  return (
+    <Flex justifyContent="space-between" flex={1} gap={3}>
+      <Flex gap={2}>
+        {icon}
+        <Typography>{message}</Typography>
+      </Flex>
+      {BoldChunk(count)}
```

Comment:
I think the `BoldChunk` was originally for passing to a translation object and I don't think it was used anywhere else. I think we should delete the component and just add the Typography bold here:
```suggestion
      <Typography fontWeight="bold">{chunks}</Typography>
```